cpp/mem.c

```diff
@@ -4,7 +4,7 @@
 #include <string.h>
 #include <stdio.h>
 
-#define MALLOC_ARENA_SIZE   32000U  /* size of initial arena fmemalloc (max 65520)*/
+#define MALLOC_ARENA_SIZE   42000U  /* size of initial arena fmemalloc (max 65520)*/
```

Comment:
we need a larger value here, as we we exploding out of memory with lots if includes in bmpview